internal/impl/kube.go

```diff
@@ -662,6 +672,9 @@ func getComponents(dep *protos.Deployment, cfg *KubeConfig) (map[string]*Replica
 		}
 	}
 
+	// Make sure weaver.Main is added.
+	components["github.com/ServiceWeaver/weaver/Main"] = &ReplicaSetConfig_Listeners{}
```

Comment:
shouldn't this be captured by the ReadComponentGraph? It feels like a hack to ensure this here.

Reply:
Yeah, it's annoying, because ReadComponentGraph returns a list of *edges*, (e.g., `Main -> Odd`), which is empty if only `Main` is present. ReadComponentGraph should probably return some kind of a `Graph` type, which includes both nodes and edges.